src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
```

Comment:
The `alt` attribute should describe the image content. Instead of 'logo', a more descriptive alternative text can be used to describe the company or site logo.